internal/mesh/internal/controllers/routes/loader/gen_memoizer_funcs.sh

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env bash
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: BUSL-1.1
```

Comment:
non-blocking: Should this be moved to a more common place so we can add it to Makefile?

Reply:
I was going to leave it for now since like `mockery` mocks, we don't currently put that into the makefile and it's a pretty specific use case.